openid-connect-server/src/main/java/org/mitre/openid/connect/config/JsonMessageSource.java

```diff
@@ -147,16 +150,14 @@ private String getValue(String code, JsonObject lang) {
 			}
 		}
 
-
 		return value;
-
 	}
 
 	/**
 	 * @param locale
 	 * @return
 	 */
-	private List<JsonObject> getLanguageMap(Locale locale) {
+	List<JsonObject> getLanguageMap(Locale locale) {
```

Comment:
Why did you make this package-private?

Reply:
so that the unit test could access that method, see here: https://github.com/mitreid-connect/OpenID-Connect-Java-Spring-Server/pull/1385/files#diff-851c9e6cde51494f569f425d1cc0211bR37

Reply:
Please don't open up the class's signature just for unit tests. I understand the importance and desire for testing this method in isolation, but if that's the case you should either make it fully public and documented or use reflection to open it specifically for testing.

Reply:
fair enough... that method already has indirect coverage via testing of the public `resolveCode()` method so i removed the tests that directly test `getLanguageMap()`